stonesoup/predictor/kalman.py

```diff
@@ -133,6 +133,34 @@ def _predict_over_interval(self, prior, timestamp):
 
         return predict_over_interval
 
+    def _predicted_covariance(self, prior, predict_over_interval, **kwargs):
+        """Private function to return the predicted covariance. Useful in that
+        it can be overwritten in children.
+
+        Parameters
+        ----------
+        prior : :class:`~.GaussianState`
+            The prior class
+        predict_over_interval : :class`~.timedelta`
+
+        Returns
+        -------
+        : :class:`~.CovarianceMatrix`
+            The predicted covariance matrix
+
+        """
+        prior_cov = prior.covar
+        trans_m = self._transition_matrix(prior=prior, time_interval=predict_over_interval,
+                                          **kwargs)
+        trans_cov = self.transition_model.covar(time_interval=predict_over_interval, **kwargs)
+
+        # As this is Kalman-like, the control model must be capable of
+        # returning a control matrix (B)
+        ctrl_mat = self._control_matrix
+        ctrl_noi = self.control_model.control_noise
+
+        return trans_m @ prior_cov @ trans_m.T + trans_cov + ctrl_mat @ ctrl_noi @ ctrl_mat.T
```

Comment:
Should group operations by order
```suggestion
        return trans_m@prior_cov@trans_m.T + trans_cov + ctrl_mat@ctrl_noi@ctrl_mat.T
```